docs/going-into-production.rst

```diff
@@ -0,0 +1,198 @@
+.. _going-into-production:
+
+=====================
+Going into production
+=====================
+
+Running CrateDB in different environments requires different approaches. This
+document outlines the basics you need to consider when going into production.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _prod-auto-boot:
+
+Bootstrapping
+=============
+
+The process of forming a cluster is known as *bootstrapping*. Consult the
+how-to guide on :ref:`CrateDB multi-node setups <multi_node_setup>` for an
+overview of the two different ways to bootstrap a cluster.
+
+If you have been using CrateDB for development on your local machine, there is
+a good chance you have been using :ref:`single host auto-bootstrapping
+<auto-bootstrapping>`.
+
+For improved performance and resiliency, production CrateDB clusters should be
+run with one node per host machine. To do this, you must manually configure the
+bootstrapping process by telling nodes how to:
+
+  a. :ref:`Discover other nodes <discovery>`, and
+  b. :ref:`Elect a master node <master-node-election>`
+
+This is known as *manual bootstrapping*. See the :ref:`how-to guide
+<manual-bootstrapping>` for more information about how bootstrap a cluster
+manually.
+
+Switching to a manual bootstrapping configuration is the first step towards
+going into production. The rest of this how-to guide covers additional things
+you may want to consider.
+
+
+.. _prod-config:
+
+Additional considerations
+=========================
+
+The following setting can be changed in your `configuration`_ file or set via
+command-line arguments. Additionally, you can change some settings at
+`runtime`_.
+
+
+.. _prod-config-path:
+
+Paths
+-----
+
+In a production environment, you should change the default locations of the
+data and log folders to avoid accidental deletion.
+
+Path configuration is especially important if you are running CrateDB on
+Docker. Any data you want to persist between container restarts should be
+located on a mounted volume.
+
+.. SEEALSO::
+
+    `Path settings`_
+
+
+.. _prod-config-cluster-name:
+
+Cluster name
+------------
+
+The `cluster.name`_ setting allows you to create multiple separate clusters. A
+node will refuse to join a cluster if the respective cluster names do not
+match.
```

Comment:
Shouldn't we change the perspective? 
Most of the users want to create a single cluster and not multiple ones.
As this is a how-to guide once again I'd add some recommendations. What name can they use? What is a good name? What not?